src/modules/users/LoginPage/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { useUser } from "../../../UserContext";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+export const Login = () => {
+  const { login } = useUser();
+  const navigate = useNavigate();
+
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+  const [error, setError] = useState("");
+
+  const handleLogin = () => {
+    const success = login(username, password);
+    if (success) {
+      navigate("/recipes");
+    } else {
+      setError("Invalid username or password!");
+    }
+  };
+
+  return (
+    <div className="entry-container">
+      <h1>Login</h1>
+      <input
```

Comment:
Make sure to add label elements to the input fields so that the user understands what to do.

Reply:
Both inputs should also be required and validated. Show error messages is unexpected value is typed in input fields.